corehq/apps/app_manager/static/app_manager/js/visit-scheduler.js

```diff
@@ -121,24 +297,22 @@ var VisitScheduler = (function () {
 
             self.hasExpiry = ko.observable();
 
-            self.hasPostSchedule = ko.observable(!!self.post_schedule_increment());
-
-            self.editValue = ko.dependentObservable({
-                read: function() {
-                    return self.post_schedule_increment();
-                },
-                write: function(newValue) {
-                    var parsedValue = parseInt(newValue, 10);
-                    this.post_schedule_increment(isNaN(parsedValue) ? newValue : parsedValue);
-                },
-                owner: self
+            self.hasRepeatVisit = ko.computed(function(){
+                return self.visits().length > 0 && self.visits()[self.visits().length - 1].type() === 'repeats';
             });
 
+            self.relevancy = ScheduleRelevancy.wrap(data.schedule);
+            var xmlRe = /\s+|<+|>+|&+|"+|'+/g;
+            self.schedule_form_id = ko.observable(data.schedule_form_id).snakeCase(xmlRe);
```

Comment:
nm, found the `snakeCase` definition. curious why you're passing a custom regex rather than stripping all non-word characters.